### DISCUSSION TITLE
Default desktop manager

### DISCUSSION THREAD

**Discussion by thek-del** [ltsp/ltsp#313] [Q&A]
I run ltsp on Linux Mint 19.3 and want to set xfce4 as the default desktop manager on clients. How can I do this?
Thanks.

**Comment by alkisg** [comment]
Are you asking how to set the default session?
It's the same as without LTSP, for example:
https://unix.stackexchange.com/questions/367294/how-do-i-change-my-default-session

**Comment by thek-del** [comment]
I mean something like LDM_SESSION="gnome-fallback" in the old ltsp.

**Comment by alkisg** [comment]
LDM was developed because LTSP5 didn't use PAM.
The new LTSP uses PAM so users can use whatever display manager they want.
So, they can configure their display managers normally, following the display manager documentation; there's no need for LTSP-specific documentation anymore.

**Comment by thek-del** [comment]
I added this in /etc/ltsp/ltsp.conf:
LIGHTDM_CONF="user-session=xfce"
And it works. 

But now this line is ignored: 
LIGHTDM_CONF="greeter-hide-users=true"

Also if i write it as:
LIGHTDM_CONF="user-session=xfce greeter-hide-users=true"

How is the correct syntax to add both lines to my clients lightdm configuration?

**Comment by alkisg** [comment]
The correct syntax is:

```shell
[clients]
LIGHTDM_CONF="user-session=xfce
greeter-hide-users=true"
```

**Comment by thek-del** [comment]
@alkisg you're the man! Worked like a charm, thank you.
I would never get by myself that a newline was required. Very unintuitive and undocumented.

**Comment by alkisg** [comment]
`man ltsp.conf` does state that it's essentially a shell script with [headers].
The string splitting then is documented in the shell documentation.

As you can see, we are very careful in assigning most documentation to other projects! :D

**Comment by thek-del** [comment]
LOL!
An example in your documentation will help the poor boys like me. :-D
